webpack.config.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
   externals: [
     nodeExternals({
       modulesFromFile: {
-        include: ['dependencies'],
+        excludeFromBundle: ['dependencies'],
```

Comment:
The original name was `include` - https://github.com/liady/webpack-node-externals/blob/v1.7.2/utils.js#L51

In 2.3.0 the `excludeFromBundle` alias was introduced: https://github.com/liady/webpack-node-externals/blame/v2.2.0/utils.js#L41

Reply:
:+1: ah, that's explain why it did work just fine (still odd choice for that option name :-P, the new option name seems better in the end, at least harder to misread)